scripts/build/python/sconstruct_helper.py

```diff
@@ -165,7 +165,7 @@ def sconscriptList(mu2eOpts):
     # If we are making a build for the trigger, do not build everything.
     if mu2eOpts["trigger"] == 'on':
         notNeeded = ["Mu2eG4/src/SConscript",
-                     "CRVResponse/src/SConscript",
+                     #"CRVResponse/src/SConscript",
```

Comment:
Why do we need to build CRVResponse for code to be shipped to the trigger.  The whole point of the notNeeded list is that we do not want the trigger builds to contain MC code and CRVResponse is a MC package.  Please let me know what you need from that package and we can discuss refactoring what lives in which directory.

Reply:
Hi Rob, CRVRsponse house MC and Reco code, that's why I need it. I agree that it would be good if someone from the CRV group would implement the separation. I remember I put this comment in a MR a long time ago, but maybe it got ignored.

Reply:
Thanks Giani.  I will speak with Ralf and cc you - I agree that he should do the work.